packages/insomnia-app/app/ui/components/settings/theme.js

```diff
@@ -114,13 +119,68 @@ class Theme extends React.PureComponent<Props, State> {
 
     return rows.map((row, i) => (
       <div key={i} className="themes__row">
-        {row.map(this.renderTheme)}
+        {row.map(theme => this.renderTheme(theme))}
       </div>
     ));
   }
 
+  renderThemeSelect(scheme: ColorScheme) {
+    const { activeLightTheme, activeDarkTheme, handleChangeTheme } = this.props;
+    const { themes } = this.state;
+
+    const activeColorTheme = {
+      light: activeLightTheme,
+      dark: activeDarkTheme,
+    }[scheme];
+
+    return (
+      <div className="form-control form-control--outlined">
+        <label>
+          Preferred {scheme} color scheme
+          <HelpTooltip className="space-left">
+            Lets you choose the color scheme that is used in {scheme} mode.
+          </HelpTooltip>
+          <select
+            value={activeColorTheme}
+            onChange={e => handleChangeTheme(e.target.value, scheme)}>
+            {themes.map(theme => (
+              <option key={theme.theme.name} value={theme.theme.name}>
+                {theme.theme.displayName}
+              </option>
+            ))}
+          </select>
+        </label>
+      </div>
+    );
+  }
+
   render() {
-    return <div className="themes pad-top">{this.renderThemeRows()}</div>;
+    const { autoDetectColorScheme, handleAutoDetectColorSchemeChange } = this.props;
+
+    return (
+      <>
+        <h2>Theme</h2>
+        <div className="themes">{this.renderThemeRows('default')}</div>
```

Comment:
```suggestion
        <div className="themes">{this.renderThemeRows()}</div>
```

Argument is now redundant 😄